Modules/Tokens/src/Tokens.Application/Tokens/Commands/CreateToken/Handler.cs

```diff
@@ -21,7 +21,7 @@ public Handler(IUserContext userContext, IMapper mapper, ITokensRepository token
 
     public async Task<CreateTokenResponse> Handle(CreateTokenCommand request, CancellationToken cancellationToken)
     {
-        var newTokenEntity = new Token(_userContext.GetAddress(), _userContext.GetDeviceId(), request.Content, request.ExpiresAt);
+        var newTokenEntity = new Token(_userContext.GetAddress(), _userContext.GetDeviceId(), request.Content, request.ExpiresAt, request.ForIdentity!);
```

Comment:
Why are you supressing null on a clearly nullable property?
Although this works with the implicit operator because `string` is a reference type, you'll end up with an actual `IdentityAddress` object where the `Value` property will be `null`.

I would suggest:
```
var forIdentity = request.ForIdentity == null ? null : IdentityAddress.Parse(request.ForIdentity);
var newTokenEntity = new Token(_userContext.GetAddress(), _userContext.GetDeviceId(), request.Content, request.ExpiresAt, forIdentity);
```